src/maud/io.py

```diff
@@ -300,10 +303,10 @@ def load_maud_input_from_toml(filepath: str, id: str = "mi") -> MaudInput:
         drain_priors=[
             Prior(
                 id=f"{dd['id']}_{e['id']}",
-                location=edd["location"],
-                scale=edd["scale"],
+                is_non_negative=False,
                 drain_id=dd["id"],
                 experiment_id=e["id"],
+                **edd,
```

Comment:
Or is there any reason to not have it like every other prior?

Reply:
I changed it as a way of getting around the fact that (depending on how the user inputs the prior) `edd` might not have keys for `"location"` and `"scale"`

Reply:
Without a location and scale drains aren't functional. It would be similar to having a reaction with no priors on parameters or enzymes.

Reply:
There should either be (a) keys for `"location"` and `"scale"` or (b) keys for 1% and 99% quantiles. These are equivalent as the quantiles later get turned into lognormal parameters. The problem is that when they are loaded the change hasn't happened yet. It's a little obscure to use the `**dict` syntax but it saves a lot of boilerplate (otherwise we would have a long `if...else` clause to run through the cases) and we have other places where we take care of validation. Also, the function `extract_priors` already uses the `**dict` method so this really just brings drains into line with the other parameters.

Reply:
Sorry for the confusion. My main concern was being able to parse drains without location and scale but this makes more sense and then do validation somewhere else. Are you still able to include a small example where you specify the input using the percentiles?